solr/core/src/java/org/apache/solr/rest/BaseSolrResource.java

```diff
@@ -147,7 +147,7 @@ protected void handleException(Logger log) {
       getSolrResponse().add("error", info);
       String message = (String) info.get("msg");
       if (null != message && !message.trim().isEmpty()) {
-        getSolrResponse().getToLog().add("msg", "{" + message.trim() + "}");
+        getSolrResponse().addToLog("msg", "{" + message.trim() + "}");
```

Comment:
Why curly brackets?  This is a symptom of code that tried to make the log parseable but it should not be the job of this class here (the caller of addToLog) to concern itself with that.